src/radix_connect/well_known_client/client.rs

```diff
@@ -0,0 +1,141 @@
+use crate::prelude::*;
+
+const SUFFIX_WELL_KNOWN_FILE: &str = ".well-known/dapp.json";
```

Comment:
add a trivial test:

`assert_eq!(SUFFIX_WELL_KNOWN_FILE, ".well-known/dapp.json")` which will capture if we accidentally change this.

Such tests ARE valuable, take 10 seconds to write but will forever protect us from accidentally changing this value!